src/app/component/Meta/Meta.component.js

```diff
@@ -19,53 +19,35 @@ import Helmet from 'react-helmet';
  */
 export default class Meta extends PureComponent {
     static propTypes = {
-        title_prefix: PropTypes.string,
-        title_suffix: PropTypes.string,
-        default_description: PropTypes.string,
-        default_keywords: PropTypes.string,
-        default_title: PropTypes.string,
-        metaObject: PropTypes.shape({
-            name: PropTypes.string,
-            meta_title: PropTypes.string,
-            meta_description: PropTypes.string,
-            meta_keyword: PropTypes.string
-        }).isRequired
-    };
-
-    static defaultProps = {
-        title_prefix: '',
-        title_suffix: '',
-        default_description: '',
-        default_keywords: '',
-        default_title: ''
+        metadata: PropTypes.arrayOf(
+            PropTypes.shape({
+                name: PropTypes.string,
+                property: PropTypes.string,
+                content: PropTypes.oneOfType([
+                    PropTypes.string,
+                    PropTypes.number
+                ])
+            })
+        ).isRequired,
+        canonical_url: PropTypes.string.isRequired,
+        title_prefix: PropTypes.string.isRequired,
+        title_suffix: PropTypes.string.isRequired,
+        title: PropTypes.string.isRequired
     };
 
     render() {
         const {
-            metaObject: {
-                name,
-                title,
-                meta_title,
-                meta_keyword,
-                meta_keywords,
-                meta_description,
-                canonical_url
-            },
-            title_prefix = '',
-            title_suffix = '',
-            default_description = '',
-            default_keywords = '',
-            default_title = ''
+            canonical_url,
+            title_prefix,
+            title_suffix,
+            metadata,
+            title
         } = this.props;
 
         return (
             <Helmet
```

Comment:
Do you think we can get rid of Helmet? It seems unreasonable now. It weights like a 1/8 of a project.

Reply:
Let's just use portal from react-dom?

Reply:
Ok! I will explore it, make relevant changes and tell you when it's done!

Reply:
It works!
I removed hardcoded title in html index templates and used portals to render
* title
* canonical url (where relevant)
* meta tags